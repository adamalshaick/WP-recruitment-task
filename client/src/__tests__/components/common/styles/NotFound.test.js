import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import {
  NotFoundWrapper,
  Button
} from "../../../../components/common/styles/NotFound";

describe("render component", () => {
  it("renders button component", () => {
    const component = renderer.create(<NotFoundWrapper />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
