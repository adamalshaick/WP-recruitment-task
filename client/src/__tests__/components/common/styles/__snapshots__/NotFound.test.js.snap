// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component renders button component 1`] = `
.c0 {
  position: absolute;
  left: 50%;
  top: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
}

<section
  className="c0"
/>
`;
